addons/mail/static/src/core/web/follower_list.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.FollowersList" owl="1">
+        <ul class="o-mail-FollowersList list-unstyled mb-0 p-2">
+            <li t-foreach="props.thread.followers" t-as="follower" t-key="follower.id">
```

Comment:
are followers can be undefined or null？

Reply:
I don't think so, it should always be an array, at least an empty one.